src/App.js

```diff
@@ -12,8 +12,8 @@ function App() {
     const [startDate, setStartDate] = useState(new Date());
     const [endDate, setEndDate] = useState(new Date());
     // Date states end
-    const [city, setCity] = useState(""); 
-    const [country, setCountry] = useState("");
+    const [city, setCity] = useState("Copenhagen"); 
+    const [country, setCountry] = useState("Denmark");
```

Comment:
This should be *reverted* before we merge :)  -- it was used to provide test data for the `Result` component.

Reply:
Fixed https://github.com/Hold-Krykke/3SemProject_FrontEnd/pull/40/commits/5c945c7918d0da54914977e1a0fa56edae47777a